### PR TITLE
Utility class border radius

### LINKED ISSUE #29989: Confusing classes `rounded-sm` and `rounded-lg`
Hello, 

I think there might be a cause of misunderstanding, which I experienced now about the `rounded-lg` and `rounded-sm` classes, because when I saw `rounded-lg`, I instantly got the message that the element edges should be rounded when the screen size is greater than `md` dimensions. 

How about using `.rounded-lg`, `.rounded-md`, `.rounded-sm`, `.rounded-xl` as they are meant to be as breakpoint definers and add `.rounded-1`, `.rounded-2`, `rounded-3` like size definitions as they are in `p-1`, `p-2` (padding) definitions?

**Use case**: For example jumbotron uses exactly this, on screens smaller than lg it has no border-radus, and larger than md, it has. Why don't we have something like this for example like the `card-img`? We can add `rounded-0 rounded-lg` to achieve this easily. `rounded-lg-1` would be delicious too.

### PR DESCRIPTION
**Pull request by JensLuyten**
Closes #29989

Updated the docs pages by searching for the `.rounded` classes and replacing them by the new `.border-radius` classes.

See 
https://deploy-preview-30048--twbs-bootstrap.netlify.com/docs/4.3/utilities/borders/#border-radius
https://deploy-preview-30048--twbs-bootstrap.netlify.com/docs/4.3/utilities/borders/#sizes

### CODE REVIEW

**Review by MartijnCuppens — DISMISSED** [review]

File: scss/_utilities.scss [hunk 1]
```diff
@@ -443,16 +443,17 @@ $utilities: map-merge(
       class: font,
       values: (monospace: $font-family-monospace)
     ),
-    "rounded": (
+    "border-radius": (
```
Comment (mdo) [inline scss/_utilities.scss]: Thoughts from @twbs/css-review for using `radius`, `br`, or something else shorter here? If we go with `br`, I’d suggest we change `border` to  ` b` as well. Good example of this is `bg` for `background-color`.

Comment (MartijnCuppens) [reply scss/_utilities.scss]: Yup, `br` and `b` make sense.

**Comment by JensLuyten** [comment]
I have shortened the new `.border-radius` class to `.br` and updated the docs with the new information.

**Comment by tpaksu** [comment]
Then "border radius bottom right" will be `br-br`, right? :)

**Comment by JensLuyten** [comment]
> Then "border radius bottom right" will be `br-br`, right? :)

The border radius class will apply to all 4 corners, so there won't be a `br-br-2` class to just set the bottom right border radius with the current implementation.

**Comment by tpaksu** [comment]
> 
> 
> > Then "border radius bottom right" will be `br-br`, right? :)
> 
> The border radius class will apply to all 4 corners, so there won't be a `br-br-2` class to just set the bottom right border radius with the current implementation.

Yes, I guess I saw the implementation in tailwindcss. That came to my mind and wanted to ask. BTW the clarification with `.border-radius` is gone with `.br`.  IMO, of course. If you want to add a line break class later for example .br { margin-bottom: 1em } this would clash. But there's also `.mb-3`, right. Well you know it better. Just my thoughts.

**Comment by MartijnCuppens** [comment]
Not sure about the `br-top` syntax, in general we concat the properties (eg. `pt` for `padding-top`). This would mean we should use `brt` instead.

Not sure about that syntax either though. @mdo, what do you think?

**Comment by ysds** [comment]
This PR does not include rename the border class name, but as @mdo commented above, the new name for border must also be considered. (This does not mean to include it in this PR.)
 
We should avoid conflicting `br` (border-radius and border-right). The best solution here is to match [Zen Cording](https://code.google.com/archive/p/zen-coding/wikis/ZenCSSPropertiesEn.wiki
). 

e.g.

* `bd-1` : `border: 1px solid $border-color`;
* `bdt-1`: `border-top: 1px solid $border-color;`
* `bdr-1`: `border-right: 1px solid $border-color;`
* `bdrs-1`: `border-radius: $border-radius-sm:`

This also compatible with the spacing classes.

**Comment by ysds** [comment]
@JensLuyten Could you change the class names to `bdrs`?

**Comment by mdo** [comment]
@ysds At that point, we might as well use `radius` instead of abbreviating :). I don't mind that fwiw.

**Comment by mdo** [comment]
@JensLuyten Could you please take a pass at updating this to reflect the latest comments? We're backpedaling on the abbreviations and are opting for `radius` as the clearest option.

**Pull request closed without merging**